### DISCUSSION TITLE
Has AzerothCore's policy regarding Blizzlike exploits changed?

### DISCUSSION THREAD

**Discussion by Santaria94** [azerothcore/azerothcore-wotlk#13902] [General]
![444](https://user-images.githubusercontent.com/32459796/202918398-9be0251b-746f-457d-9c92-9b20467dfd54.jpg)

Blizzard confirm that it was an exploit and fix it in Wrath Classic: https://www.wowhead.com/wotlk/news/wrath-classic-hotfixes-october-14-deep-wounds-and-fiery-weapon-329368

AzerothCore...
https://github.com/azerothcore/azerothcore-wotlk/pull/13494

**Comment by Nyeriah** [comment]
Hello,

It is not clear to us if Blizzard has classified this as an exploit or rather an uintended game mechanic that they corrected later on. But about this case in particular, there's a conflict that we will probably solve on a case by case basis: we will not be making Classic WotLK changes at Azerothcore.

So to answer your question, this PR was accepted because: 1) it is unclear that it was an exploit so to speak or just a badly designed mechanic 2) the fix to such mechanic only came with Classic and we are not looking forward Classic content.

Regards.

**Reply by Plasmashield** [reply]
> Hello,
> 
> It is not clear to us if Blizzard has classified this as an exploit or rather an uintended game mechanic that they corrected later on. But about this case in particular, there's a conflict that we will probably solve on a case by case basis: we will not be making Classic WotLK changes at Azerothcore.
> 
> So to answer your question, this PR was accepted because: 1) it is unclear that it was an exploit so to speak or just a badly designed mechanic 2) the fix to such mechanic only came with Classic and we are not looking forward Classic content.
> 
> Regards.

Now let's look at this again and make it clear:
- We have a mechanic that no one knew about before the wotlk classic
- People, having seen it on the wotlk classic, whining for you to implement it on the AC to abuse this thing
- You can see that on wotlk classic this led to undesirable consequences and Blizzard decides to remove this mechanic and very clearly explain their position and the possible damage to the game from using this mechanic
- For some obscure reason, you're ignoring Blizzard, ignoring their position on the undesirable consequences of this mechanic and implement it for only one purpose - to let people who know about it abuse this mechanic

So...What's the point?

**Comment by virgo77** [comment]
don't forget that the "classic" versions of WoW (Vanilla, BC, WotLK) are based on the BNET Client. So it has been recoded to work on it.
The original clients of these games versions are no longer existing anymore from Blizzard, but you can still download them from some websites. So maybe that the fix they had to apply in WotLK Classic only exists because it relies on the BNet Client.

And our great devs here are working on a full great WotLK orginal server version (3.3.5a build 12340). They're not here for fixing bugs that were potentialy existing around 2008/2010.

Kind Regards!

V.
